packages/sync/src/modules/Callables.php

```diff
@@ -38,6 +38,33 @@ class Callables extends Module {
 	 */
 	private $callable_whitelist;
 
+	/**
+	 * For some options, we should always send the change right away!
+	 *
+	 * @access public
+	 *
+	 * @var array
+	 */
+	const ALWAYS_SEND_UPDATES_TO_THESE_OPTIONS = array(
+		'jetpack_active_modules',
+		'home', // option is home, callable is home_url.
+		'siteurl',
+		'jetpack_sync_error_idc',
+		'paused_plugins',
+		'paused_themes',
+	);
+
+	/**
+	 * For some options, the callable key differs from the option name/key
+	 *
+	 * @access public
+	 *
+	 * @var array
+	 */
+	const OPTION_NAMES_TO_CALLABLE_NAMES = array(
+		'home' => 'home_url',
```

Comment:
Any others we should take care for? For example, unless I'm wrong, I think `paused_themes` and `paused_plugins` should correspond to `wp_paused_themes()` and `wp_paused_plugins`. Should we do a full audit of the "always send updates to these options" corresponding callables?

Reply:
@tyxla , the goal of this PR was to ensure that changing the `home` option via WP-CLI could sync, https://github.com/Automattic/jetpack/issues/12924, so that's what I tested / focused on. I haven't explored other options/callables, and would like to get this PR merged. What about exploring additional options/callables in a subsequent PR or adding them as necessary?

Reply:
Another PR sounds good to me! I'd be fine with adding a `@TODO` in the docblock or inside the array so we keep track of what needs to happen here.